internal/common/filespec.go

```diff
@@ -47,6 +48,11 @@ func (this *FileSpec) Validate() error {
 		if this.Path == "" {
 			return fmt.Errorf("A path FileSpec cannot have an empty path.")
 		}
+
+		_, err := filepath.Glob(this.Path)
```

Comment:
This is not good.
Although it technically works, it is excessive (and a small potential security issue).

First thing to note is that FileSpecs can be validated at any time, but that is not when they are executed/used (often inside of a Docker container).
Additionally many file specs can (and are required to be) relative paths.
So here you have no idea what paths is being traversed.
Although filepath.Glob() only returns errors on bad validation, you can be using a lot of time and memory traversing paths.
(This is where the security issue comes in, since this information could be used in a timing attack.)

Instead, go to the source and see how filepath.Glob() validates.